lasagne/tests/layers/test_noise.py

```diff
@@ -28,6 +28,11 @@ def layer_p_02(self, input_layer):
         from lasagne.layers.noise import DropoutLayer
         return DropoutLayer(input_layer, p=0.2)
 
+    def test_float64_leaks(self, input_layer) :
+        from lasagne.layers.noise import DropoutLayer
+	theano.config.floatX = 'float32'
+	assert DropoutLayer(input_layer,p=numpy.float32(0.5)).get_output_for(theano.shared(numpy.ones((100,100),dtype=numpy.float32))).eval().dtype is numpy.dtype('float32')
```

Comment:
The indentation is wrong, you're missing a space for the second and third line. In addition, the third line is much too long (the limit is 79 characters).
The test is not good either, but you couldn't know: When you set `theano.config.floatX = 'float32'`, this will affect all future tests. You could save the previous value and restore it (after obtaining the dtype of the result, but before asserting it is float32).